.travis/ci_script.bash

```diff
@@ -4,10 +4,11 @@ set -e
 cd $ROS2_WS
 
 # install dependencies
-apt-get -qq update && rosdep install -y \
+rosdep update && apt-get -qq update && rosdep install -y \
   --from-paths src \
   --ignore-src \
-  --rosdistro $ROS_DISTRO
+  --rosdistro $ROS_DISTRO \
+  --skip-keys sros2_cmake
```

Comment:
Note to self: revert this workaround once https://github.com/ros2/sros2/pull/101 is merged